src/omv/modules/py_tf.h

```diff
@@ -26,4 +27,15 @@ extern size_t py_tf_putchar_buffer_index;
 extern size_t py_tf_putchar_buffer_len;
 void py_tf_alloc_putchar_buffer();
 
+// Functionality select
+#if IMLIB_ENABLE_TF == IMLIB_TF_FULLOPS
+#define libtf_get_parameters libtf_all_ops_get_parameters
+#define libtf_invoke libtf_all_ops_invoke
+#elif IMLIB_ENABLE_TF == IMLIB_TF_DEFAULT
+#define libtf_get_parameters libtf_reduced_ops_get_parameters
+#define libtf_invoke libtf_reduced_ops_invoke
+#elif IMLIB_ENABLE_TF == IMLIB_TF_MINIMAL
+#define libtf_get_parameters libtf_minimum_ops_get_parameters
+#define libtf_invoke libtf_minimum_ops_invoke
+#endif
```

Comment:
While we're at it, I would prefer if the names match the options.

```suggestion
#if IMLIB_ENABLE_TF == IMLIB_TF_FULLOPS
#define libtf_get_parameters libtf_get_parameters_fullops
#define libtf_invoke libtf_invoke_fullops
#elif IMLIB_ENABLE_TF == IMLIB_TF_DEFAULT
#define libtf_get_parameters libtf_get_parameters_default
#define libtf_invoke libtf_invoke_default
#elif IMLIB_ENABLE_TF == IMLIB_TF_MINIMAL
#define libtf_get_parameters libtf_get_parameters_minimal
#define libtf_invoke libtf_invoke_minimal
#endif
```